.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+* @adutra @annafil @anoopj @ashvina @dennishuo @dimas-b @eric-maynard @jackye1995 @jbonofre @vvcephei @collado-mike @snazy @RussellSpitzer @takidau @MonkeyCanCode @flyrain @ebyhr
```

Comment:
It seems `@annafil` `@anoopj` are invalid GitHub handles?

Reply:
`@annafil` looks correct: https://github.com/annafil
`@anoopj` also looks valid :  https://github.com/anoopj` (but IDK if it's the intentended person :) )

Reply:
Maybe GH flags these people because of write permissions to this repo?